purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -110,6 +111,13 @@ internal class PurchasesOrchestrator(
             purchasesStateCache.purchasesState = value
         }
 
+    val latestConfiguration: PurchasesConfiguration
+        get() = if (initialConfiguration.appUserID == null) {
```

Comment:
I had to add this condition... Basically, the condition to avoid reconfiguration was never wrong if the developer doesn't set an `appUserId`. In this case, we should just reuse the last configured user id, so it should be safe.